so-widgets-bundle.php

```diff
@@ -351,6 +356,59 @@ function admin_ajax_manage_handler(){
 	}
 
 	/**
+	 *
+	 */
+	function admin_ajax_settings_form(){
+		$widget_objects = $this->get_widget_objects();
+		$widget_object = !empty( $widget_objects[ $_GET['id'] ] ) ? $widget_objects[ $_GET['id'] ] : false;
+
+		if( empty( $widget_object ) ) exit();
+
+		$form_options = $widget_object->initialize_settings_form();
```

Comment:
Seems a bit weird to get the `$form_options` from the `$widget_object` and then pass it back in further down the line, without actually doing anything to `$form_options`. Here a method for `has_gloabl_settings()` would be useful again.